lib/src/model/user.dart

```diff
@@ -223,16 +220,28 @@ class UserLinks extends ModelBase {
     };
   }
 
-  factory UserLinks.fromJson(Map<String, dynamic> json) {
+  factory UserLinks.fromJson(Map<String, dynamic>? json) {
+    if (json == null) {
+      return UserLinks(
+        self: Uri(),
+        html: Uri(),
+        photos: Uri(),
+        likes: Uri(),
+        portfolio: Uri(),
+        followers: Uri(),
+        following: Uri(),
+      );
+    }
+
     return UserLinks(
       source: json,
-      self: (json['self'] as String).let(Uri.parse),
-      html: (json['html'] as String).let(Uri.parse),
-      photos: (json['photos'] as String).let(Uri.parse),
-      likes: (json['likes'] as String).let(Uri.parse),
-      portfolio: (json['portfolio'] as String).let(Uri.parse),
-      followers: (json['followers'] as String).let(Uri.parse),
-      following: (json['following'] as String).let(Uri.parse),
+      self: (json['self'] as String?)?.let(Uri.parse) ?? Uri(),
+      html: (json['html'] as String?)?.let(Uri.parse) ?? Uri(),
+      photos: (json['photos'] as String?)?.let(Uri.parse) ?? Uri(),
+      likes: (json['likes'] as String?)?.let(Uri.parse) ?? Uri(),
+      portfolio: (json['portfolio'] as String?)?.let(Uri.parse) ?? Uri(),
+      followers: (json['followers'] as String?)?.let(Uri.parse) ?? Uri(),
```

Comment:
The exception is thrown when trying to parse `followers`:

https://github.com/blaugold/unsplash_client/blob/e29ab00b1e4a80e79ebd6e6b2db9e720cb02491a/lib/src/model/user.dart#L234

Can we just make `followers` nullable in `UserLinks`?

I would rather make a breaking change than return empty `Uri`s or `UserLinks` without any links.